pkg/agent/util/net.go

```diff
@@ -405,3 +409,50 @@ func GetGlobalIPNetsByName(link *net.Interface) ([]*net.IPNet, error) {
 	}
 	return addrs, nil
 }
+
+func GenUplinkInterfaceName(hostIfName string) string {
+	return fmt.Sprintf("%s%s", PrefixPhy, hostIfName)
+}
+
+func GetUplinkConfig(uplinkIfName string) (*config.AdapterNetConfig, error) {
```

Comment:
Could you please recheck this part? The reason I put it and RenameInterface below into net_linux.go previously is because I find it can cause compile issue for antctl-darwin if we put it here. We may need to have separate implementations for Linux and Windows.

`# antrea.io/antrea/pkg/agent/util
Error: pkg/agent/util/net.go:407:21: undefined: GetNetRoutesOnAdapter
Error: pkg/agent/util/net.go:442:15: undefined: renameHostInterface
make: *** [Makefile:188: antctl-darwin] Error 2
Error: Process completed with exit code 2.`

Reply:
yes. moving the function to net.go will give undefined errors for `renameHostInterface` and `GetNetRoutesOnAdapter`. @wenyingd  Do you think it's good idea to move these functions  into net_linux/net_windows, on the downside there will be duplicated code.